src/firebase.js

```diff
@@ -14,14 +14,32 @@ const firebaseConfig = {
 };
 
 const app = initializeApp(firebaseConfig);
-export const db = getFirestore(app)
-export const auth = getAuth(app);
 
 if (process.env.NODE_ENV === "development") {
 	self.FIREBASE_APPCHECK_DEBUG_TOKEN = process.env.REACT_APP_APP_CHECK_DEBUG_TOKEN ?? true;
 } else if (process.env.NODE_ENV === "test") {
 	self.FIREBASE_APPCHECK_DEBUG_TOKEN = process.env.REACT_APP_APP_CHECK_DEBUG_TOKEN;
 }
+
+let db;
+let auth;
+
+if (process.env.REACT_APP_USE_EMULATOR === "true") { // run this with npm run start:emulators
+	db = getFirestore();
+	auth = getAuth();
+	connectAuthEmulator(auth, "http://127.0.0.1:9099");
+	connectFirestoreEmulator(db, '127.0.0.1', 8080);
+	console.log('Using emulators');
+}
+else { // run this with just npm start
+	db = getFirestore(app);
+	auth = getAuth(app);
+	console.log('Using normal');
```

Comment:
Do we want to send a console message in the production version? Just a simple console message when we run using emulators should be enough, right?

Reply:
Yeah get rid of line 37